common/components/ui/OfflineSymbol.tsx

```diff
@@ -26,7 +26,7 @@ const OfflineSymbol = ({ offline, size }: OfflineSymbolProps) => {
       break;
   }
 
-  return <img src={offline ? wifiOff : wifiOn} width={width} height={height} />;
+  return <img src={offline ? wifiOff : wifiOn} alt="wifi status" width={width} height={height} />;
```

Comment:
We actually don't use this component anywhere anymore, it can be removed entirely.